src/lib/eslint/TypescriptEslintStrategy.ts

```diff
@@ -67,16 +68,18 @@ export class TypescriptEslintStrategy implements EslintStrategy {
 			.replace('index.js', path.join('configs', 'eslint-recommended.js'));
 		// The extended base config has an `overrides` property which is a singleton array of the rule configs object
 		// we actually want.
-		const rawExtendedConfig = (await import(pathToExtendedBaseConfig)) as {overrides: ESRuleConfig[]};
-		this.extendedEslintConfig = rawExtendedConfig.overrides[0];
+		const rawExtendedConfig = (await import(pathToFileURL(pathToExtendedBaseConfig).href)) as {default: {overrides: ESRuleConfig[]}};
```

Comment:
Taking a while to get this to work on windows with the new node16 module loader which has much stricter requirements. Hopefully this will work (got the idea from https://stackoverflow.com/questions/77980517/idiot-developer-unable-to-import-files-dynamically-in-nodejs-with-esm-modules)... crossing my fingers that with this the windows tests will pass.

Previously it failed with: https://github.com/forcedotcom/sfdx-scanner/actions/runs/8287401013/job/22679738109?pr=1396